docs/project/olmv1_community.md

```diff
@@ -0,0 +1,11 @@
+In this next iteration of OLM, the community has also taken care to make it as contributor-friendly as possible, and welcomes new contributors.
+The project is tracked in a [GitHub project](https://github.com/orgs/operator-framework/projects/8/),
```

Comment:
Do we need to note that this is a CNCF project or is that self-evident?

Reply:
Added ^^